testnet-contracts/package.json

```diff
@@ -24,12 +28,14 @@
     "chai-bignumber": "^3.0.0",
     "openzeppelin-solidity": "^2.1.3",
     "solc": "^0.5.0",
-    "web3": "^1.0.0-beta.52",
+    "web3": "^1.2.1",
     "web3-utils": "^1.0.0-beta.52"
   },
   "scripts": {
     "develop": "truffle develop",
-    "test": "truffle test",
-    "exit": ".exit"
+    "migrate": "truffle compile && truffle migrate --reset --all",
+    "peggy:address": "truffle exec scripts/getDeployedAddress.js",
+    "peggy:lock": "truffle exec scripts/sendLockTx.js",
+    "peggy:abi": "solc --abi ./contracts/Peggy.sol -o ../cmd/ebrelayer/contract/abis --overwrite"
```

Comment:
why does this use solc and not just truffle?

Reply:
if we need to continue using solc, can you also specify in requirements which solc (eg: from https://solidity.readthedocs.io/en/v0.5.3/installing-solidity.html, you see the command is different when installing from npm than from apt-get, ie, solcjs vs solc)

Reply:
removed solc https://github.com/swishlabsco/peggy/pull/2/commits/527279c23994a5a13546f05b2f87caedcd598c77